package.json

```diff
@@ -51,14 +51,17 @@
   "scripts": {
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
-    "test": "jest --silent --env=jsdom",
+    "test": "jest --env=jsdom",
```

Comment:
What changed?

Reply:
You were right, haha! the silent flag hid the failing tests. Instead, i added stuff to the jest config that lets me run the polyfill without running CRA test